bin/src/Flags.purs

```diff
@@ -306,3 +306,10 @@ ensureRanges =
     ( O.long "ensure-ranges"
         <> O.help "Add version bounds for all the dependencies of the selected project"
     )
+
+sourceMap :: Parser Boolean
+sourceMap =
+  O.switch
+    ( O.long "source-map"
```

Comment:
Technically the command creates a single source map, but the old spago command was plural. `--source-maps`. The esbuild flag is singular though.

Reply:
I'd prefer the plural version,`purs` uses that